### ISSUE TITLE
Tarpaulin exit with code 0 when failed to run tests

### ISSUE THREAD

**Issue by breeswish** [xd009642/tarpaulin#221]
Looks like Tarpaulin does not return an exit code indicating failure even if there are indeed errors:

```
sh-4.2$ cargo tarpaulin -v
[INFO tarpaulin] Running Tarpaulin
[INFO tarpaulin] Building project
    Finished dev [unoptimized + debuginfo] target(s) in 0.31s
[DEBUG tarpaulin] Processing tikv
[INFO tarpaulin] Launching test
………………
[ERROR tarpaulin] Failed to get test coverage! Error: Failed to run tests! Error: A segfault occurred while executing tests
sh-4.2$ echo $?
0
```

**Comment by breeswish** [comment]
The error should be a known issue https://github.com/xd009642/tarpaulin/issues/190 :)

**Comment by xd009642** [comment]
Yeah exit codes are something where I'm going to have to go over them all and figure out what behaviour I want for each one!

**Comment by AnderEnder** [comment]
@xd009642 ,
At least it should be not zero now(which is a main requested behaviour in CI services), and possible exit codes behaviour could be extended later :).

**Comment by xd009642** [comment]
This is now fixed and in 0.8.5 :+1:
